internal/postgres/postgres.go

```diff
@@ -64,3 +65,35 @@ func buildConnInfo(c Pgconf) string {
 
 	return connInfo
 }
+
+// GetHeader retrieves the file header
+func GetHeader(db *sql.DB, fileID int) ([]byte, error) {
+	const getHeader = "SELECT header from local_ega.files WHERE id = $1"
+
+	var hexString string
+	if err := db.QueryRow(getHeader, fileID).Scan(&hexString); err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	header, err := hex.DecodeString(hexString)
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	return header, nil
+}
+
+// MarkCompleted markes the file as "COMPLETED"
+func MarkCompleted(db *sql.DB, checksum string, fileID int) (error) {
+	const completed = "UPDATE local_ega.files SET status = 'COMPLETED', archive_file_checksum = $1, archive_file_checksum_type = 'SHA256'  WHERE id = $2;"
+	res, err := db.Exec(completed, checksum, fileID)
+	if err != nil {
+		log.Errorf("something went wrong with the DB qurey: %s", err)
+	}
+	if ra, _ := res.RowsAffected(); ra == 0 {
```

Comment:
Sorry for nitpicking, but, again, `ra`, res`? Can we have it more detailed? It's clear that you know the meaning while you are writing the code, but when others are reading it, we actually have to jump to the declaration of the variable to see that `ra` is actually `RowsAffected`. Why not just call it `rowsAffected`?